README.md

```diff
@@ -40,6 +40,12 @@ grunt
 cordova platform add browser
 cordova run browser
 ```
+
+If you have the error "Error executing "google-chrome --user-data-dir=/tmp/temp_chrome_user_data_dir_for_cordova http://localhost:8000/index.html": /bin/sh: 1: google-chrome: not found" you can use the command below to run in the browser:
+
+```
+cordova run browser -- --port=8001 --target=Chromium

```

Comment:
I think the `--port=8001` is only necessary if there is a port conflict. The default port 8000 should be fine for most people. The error you cited here is caused by cordova not finding chrome.